packages/mapviewer/src/utils/components/SimpleWindow.vue

```diff
@@ -78,8 +78,9 @@ onMounted(() => {
     if (movable) {
         const windowElement = windowRef.value
         const headerElement = headerRef.value
-        useMovableElement(windowElement, {
-            grabElement: headerElement,
+        useMovableElement({
+            element: windowElement.value,
+            grabElement: headerElement.value,
```

Comment:
thos are already the values of windowRef or headerRef, no need for an extra value here.